system-tests/test/component_testing_index_html_spec.ts

```diff
@@ -0,0 +1,29 @@
+import systemTests from '../lib/system-tests'
+
+describe('component testing projects', function () {
+  systemTests.setup()
+
+  systemTests.it('create-react-app-custom-index-html', {
+    project: 'create-react-app-custom-index-html',
+    testingType: 'component',
+    spec: 'src/App.cy.jsx',
+    browser: 'chrome',
+    expectedExitCode: 0,
+  })
+
+  systemTests.it('vueclivue3-custom-index-html', {
+    project: 'vueclivue3-custom-index-html',
+    testingType: 'component',
```

Comment:
Definitely like these system tests, but we also should keep an eye on how much time they add to CI (if they do) and be conscious of it.

Reply:
May be time to up the `parallelization` on these jobs in `circle.yml`.